lua/lspconfig/server_configurations/ols.lua

```diff
@@ -4,8 +4,8 @@ return {
   default_config = {
     cmd = { 'ols' },
     filetypes = { 'odin' },
-    root_dir = util.root_pattern('ols.json', '.git'),
-    single_file_support = true,
+    root_dir = util.root_pattern('ols.json', '.git', '*.odin'),
+    single_file_support = false,
```

Comment:
if it not support remove.

Reply:
I went ahead and removed `single_file_support` since it seems to default to false. :+1: